src/test/ui/ffi_const.rs

```diff
@@ -0,0 +1,6 @@
+// ignore-tidy-linelength
+#![feature(ffi_const)]
+#![crate_type = "lib"]
+
+#[ffi_const] //~ ERROR `#[ffi_const]` may only be used on `extern fn`s [E0725]
```

Comment:
Hmm, this seems misleading - it's talking about an `fn` inside an `extern` block, not an `extern fn`, which is something different

Reply:
Would changing this to "may only be used on foreign functions" make it more clear?

Reply:
Yeah, that sound better (unless someone has a better idea).

Reply:
So I've changed it to foreign functions :)